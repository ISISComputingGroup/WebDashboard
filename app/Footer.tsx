import React from "react";
import Link from "next/link";
import Image from "next/image";

export default function Footer() {
  return (
    <section className="bg-white dark:bg-zinc-900  ">
      <div className="max-w-screen-xl px-4 py-12 mx-auto space-y-8 overflow-hidden sm:px-6 lg:px-8 w-full flex flex-col justify-center items-center ">
        <Link href="/" className="flex items-center justify-center ">
          <Image
            src="/WebDashboard/IBEX_logo.png"
            alt="IBEX Logo"
            className="h-16 w-auto"
            width={128}
            height={128}
          />
        </Link>
        <nav className="flex flex-wrap justify-center -mx-5 -my-2">
          <Link
            href="#"
            className="text-base leading-6 text-gray-500 hover:text-gray-900  dark:text-white dark:hover:text-gray-400 transition-color duration-100 px-5 py-2"
          >
            Home
          </Link>
          <Link
            href="/instruments"
            className="text-base leading-6 text-gray-500 hover:text-gray-900  dark:text-white dark:hover:text-gray-400 transition-color duration-100 px-5 py-2"
          >
            Instrument statuses
          </Link>

          <Link
            href="/wall"
            className="text-base leading-6 text-gray-500 hover:text-gray-900  dark:text-white dark:hover:text-gray-400 transition-color duration-100 px-5 py-2"
          >
            The Wall
          </Link>

          <Link
            href="#"
            className="text-base leading-6 text-gray-500 hover:text-gray-900  dark:text-white dark:hover:text-gray-400 transition-color duration-100 px-5 py-2"
          >
            Terms
          </Link>
        </nav>
        <p className="mt-8 text-base leading-6 text-center text-gray-400">
          Â© 2024 Experiment Controls at ISIS. All rights reserved.
        </p>
      </div>
    </section>
  );
}
