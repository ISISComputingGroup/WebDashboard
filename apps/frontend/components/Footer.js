import React from "react";
import Link from "next/link";

export default function Footer() {
  return (
    <section className="bg-white dark:bg-zinc-900 border-t ">
      <div className="max-w-screen-xl px-4 py-12 mx-auto space-y-8 overflow-hidden sm:px-6 lg:px-8 w-full flex flex-col justify-center items-center ">
        <Link href="/" className="flex items-center justify-center ">
          <img src="/ibex_logo.png" alt="IBEX Logo" className="h-16 w-auto" />
        </Link>
        <nav className="flex flex-wrap justify-center -mx-5 -my-2">
          <Link
            href="#"
            className="text-base leading-6 text-gray-500 hover:text-gray-900  dark:text-white dark:hover:text-gray-400 transition-color duration-100 px-5 py-2"
          >
            Home
          </Link>
          <Link
            href="/instruments"
            className="text-base leading-6 text-gray-500 hover:text-gray-900  dark:text-white dark:hover:text-gray-400 transition-color duration-100 px-5 py-2"
          >
            Instrument statuses
          </Link>

          <Link
            href="/wall"
            className="text-base leading-6 text-gray-500 hover:text-gray-900  dark:text-white dark:hover:text-gray-400 transition-color duration-100 px-5 py-2"
          >
            The Wall
          </Link>

          <Link
            href="/support"
            className="text-base leading-6 text-gray-500 hover:text-gray-900  dark:text-white dark:hover:text-gray-400 transition-color duration-100 px-5 py-2"
          >
            Support
          </Link>

          <Link
            href="#"
            className="text-base leading-6 text-gray-500 hover:text-gray-900  dark:text-white dark:hover:text-gray-400 transition-color duration-100 px-5 py-2"
          >
            Terms
          </Link>
        </nav>
        <p className="mt-8 text-base leading-6 text-center text-gray-400">
          Â© 2024 Experiment Controls at ISIS. All rights reserved.
        </p>
      </div>
    </section>
  );
}
