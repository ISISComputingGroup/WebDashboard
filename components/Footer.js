import React from "react";
import Link from "next/link";

export default function Footer() {
  return (
    <section className="bg-white">
      <div className="max-w-screen-xl px-4 py-12 mx-auto space-y-8 overflow-hidden sm:px-6 lg:px-8">
        <nav className="flex flex-wrap justify-center -mx-5 -my-2">
          <div className="px-5 py-2">
            <Link
              href="#"
              className="text-base leading-6 text-gray-500 hover:text-gray-900"
            >
              Instruments
            </Link>
          </div>
          <div className="px-5 py-2">
            <Link
              href="#"
              className="text-base leading-6 text-gray-500 hover:text-gray-900"
            >
              How to use
            </Link>
          </div>
          <div className="px-5 py-2">
            <Link
              href="#"
              className="text-base leading-6 text-gray-500 hover:text-gray-900"
            >
              Terms
            </Link>
          </div>
        </nav>
        <p className="mt-8 text-base leading-6 text-center text-gray-400">
          Â© 2021 Experiment Controls at ISIS. All rights reserved.
        </p>
      </div>
    </section>
  );
}
