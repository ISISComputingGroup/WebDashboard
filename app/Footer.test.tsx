import { render } from "@testing-library/react";
import Footer from "@/app/Footer";

it("renders footer unchanged", () => {
  const { container } = render(<Footer />);
  expect(container).toMatchSnapshot();
});
