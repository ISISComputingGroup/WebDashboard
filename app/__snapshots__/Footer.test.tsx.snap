// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer unchanged 1`] = `
<div>
  <footer
    class="bg-white dark:bg-zinc-900  "
  >
    <div
      class="max-w-screen-xl px-4 py-12 mx-auto space-y-8 overflow-hidden sm:px-6 lg:px-8 w-full flex flex-col justify-center items-center "
    >
      <a
        class="flex items-center justify-center "
        href="/"
      >
        <img
          alt="IBEX Logo"
          class="h-16 w-auto"
          data-nimg="1"
          decoding="async"
          height="128"
          loading="lazy"
          src="/_next/image?url=%2FWebDashboard%2FIBEX_logo.png&w=256&q=75"
          srcset="/_next/image?url=%2FWebDashboard%2FIBEX_logo.png&w=128&q=75 1x, /_next/image?url=%2FWebDashboard%2FIBEX_logo.png&w=256&q=75 2x"
          style="color: transparent;"
          width="128"
        />
      </a>
      <nav
        class="flex flex-wrap justify-center -mx-5 -my-2"
      >
        <a
          class="text-base leading-6 text-gray-500 hover:text-gray-900  dark:text-white dark:hover:text-gray-400 transition-color duration-100 px-5 py-2"
          href="/"
        >
          Home
        </a>
        <a
          class="text-base leading-6 text-gray-500 hover:text-gray-900  dark:text-white dark:hover:text-gray-400 transition-color duration-100 px-5 py-2"
          href="/instruments"
        >
          Instrument statuses
        </a>
        <a
          class="text-base leading-6 text-gray-500 hover:text-gray-900  dark:text-white dark:hover:text-gray-400 transition-color duration-100 px-5 py-2"
          href="https://www.isis.stfc.ac.uk/Pages/Beam-Status.aspx"
          target="_blank"
        >
          Beam status
        </a>
      </nav>
    </div>
  </footer>
</div>
`;
